app/components/search/CategoriesRefinementList.jsx

```diff
@@ -67,29 +67,42 @@ class CategoriesRefinementList extends Component {
     }
   }
 
+  refinementHasResults(key) {
+    const { items } = this.props;
+    return items.find(item => item.label === key);
+  }
+
   render() {
     const { isChecked } = this.state;
     const mapKeys = Object.keys(this.categoriesMapping);
     return (
       <div className="refinement-wrapper">
         <p className="refinement-title">Categories</p>
         <ul className="refinement-ul">
-          {mapKeys.map(key => (
+          {mapKeys.map(key => {
+            const refinementHasResults = this.refinementHasResults(key);
             // for each map key, display it as a filtering option
             // for onClick of each option, call refine on the values of the key
             // eslint-disable-next-line prefer-template
-            <li key={key} className={'refine-li ' + (isChecked[key] ? 'active' : '')}>
-              <label>
-                <input
-                  type="checkbox"
-                  className="refine-checkbox"
-                  onChange={this.changeRefinement.bind(this, key)}
-                  checked={isChecked[key]}
-                />
-                {key}
-              </label>
-            </li>
-          ))}
+            return (
+              <li
+                key={key}
+                className={`refine-li ${isChecked[key] ? 'active' : ''
+                }${!refinementHasResults ? 'refine-li-disabled' : ''}`}
```

Comment:
Minor: I'd suggest breaking out this logic for computing the CSS class names into pure JS, since that might make it easier to read.